src/extract.js

```diff
@@ -504,45 +505,28 @@ exports.Extractor = class Extractor {
 
     if (filename.endsWith('.xml')) {
       const textArray = [];
-      this.options.xmlAttributes.map((keyword) => {
-        if (keyword === 'text' && node.html()) {
-          let element;
-          if (node[0].name === 'text') {
-            node[0].children.forEach(child => {
-              if (child.name === 'p') {
-                element = node.html().substring(
-                  node.html().indexOf('>') + 1,
-                  node.html().lastIndexOf('</p>'),
-                );
-              }
-            });
-          }
-
-          if (element) {
-            if (!element.trim().startsWith('{{') && !element.trim().endsWith('}}')) {
-              textArray.push(element);
-            }
-          }
-        } else if (node.attr(keyword) !== undefined) {
-          const value = node.attr(keyword);
-          if (!value.trim().startsWith('{{') && !value.trim().endsWith('}}')) {
-            textArray.push(node.attr(keyword));
-          } else if (value.trim().startsWith('{{') && value.trim().endsWith('}}')) {
-            const exp = new RegExp('(?<=\').+?(?=\')', 'g');
-            const matches = value.match(exp);
-            if (matches) {
-              matches.forEach(match => {
-                const finalMatch = match.replace(/\\/, '');
-                textArray.push(finalMatch);
-              });
-            }
-          }
+      const element = node[0];
+      const text = element.data ? element.data.trim() : '';
+      const findTextsInExpression = (value) => value.match(new RegExp("(?<=').+?(?=')", 'g')) || [];
```

Comment:
@digi604 this is our only remaining problem right now because the `RegEx` is not 100% correct and the pot output contain values such as
```
#: app.xml
msgid "+ DATE(currentUser.Last_Execution_Date__c) : "
msgstr ""

#: app.xml
msgid "+DATE(coaching.PCH__Execution_Date__c) :"
msgstr ""
```

Maybe it should be handled in a different PR? WDYT?